lib/active_force/sobject.rb

```diff
@@ -193,6 +197,14 @@ def []=(name,value)
       send("#{name.to_sym}=", value)
     end
 
+    def save_request
```

Comment:
For this feature, we only really need `body`.  However, this will be a necessary method when implementing other Composite API features like [Composite Graph](https://developer.salesforce.com/docs/atlas.en-us.api_rest.meta/api_rest/resources_composite_graph.htm).